server/tests/controller/userFollow/follow.spec.js

```
@@ -0,0 +1,135 @@
+      const res = await chai
+        .request(app)
+        .post(`/api/v1/profiles/${followerId}/follow`)
+        .set({ Authorization: token });
+      expect(res.statusCode).to.equal(403);
+      expect(res.body.success).to.be.false;
+      expect(res.body.message).to.equal("You cannot follow yourself");
+    });
+
+    it("User should be able to unfollow an author", async () => {
+      const users = await userDependencies();
+      const followerId = users[0].id;
+      const authorId = users[1].id;
+      await Followers.create({ authorId, followerId });
+      const token = generateJWT({ userId: followerId }, jwtConfigs);
+      const res = await chai
+        .request(app)
+        .post(`/api/v1/profiles/${authorId}/follow`)
+        .set({ Authorization: token });
+      expect(res.statusCode).to.equal(200);
+      expect(res.body.success).to.be.true;
+      expect(res.body.message).to.equal(
+        "You are no longer following John Drew"
+      );
+    });
+  });
+
+  describe("GET </api/v1/profiles/follow>", () => {
+    it("User should be able to see all authors followed", async () => {
+      const users = await userDependencies();
+      const followerId = users[0].id;
+      const author1Id = users[1].id;
+      const author2Id = users[2].id;
+      await Followers.bulkCreate([
+        { authorId: author1Id, followerId },
+        { authorId: author2Id, followerId }
+      ]);
+      const token = generateJWT({ userId: followerId }, jwtConfigs);
+      const res = await chai
+        .request(app)
+        .get(`/api/v1/profiles/follow`)
+        .set({ Authorization: token });
+      expect(res.statusCode).to.equal(200);
+      expect(res.body.success).to.be.true;
+      expect(res.body.message).to.equal("Followed authors retrieved");
+      expect(res.body.count).to.equal(2);
+      expect(res.body.data).to.be.an("array");
+    });
+
+    it("Should return you are not following any author if none is followed", async () => {
```

`should return an error .....` This test here is not descriptive enough